RELEASES.md

```diff
@@ -0,0 +1,131 @@
+# 🚀 Releases
+
+This document outlines the Moby project’s release process, including cadence and versioning.
+
+## 📌 Versioning
+
+Releases of moby modules will be versioned using dotted triples, similar to
```

Comment:
It's a module, so not similar to, it _must_ be SemVer (its part of the contract of using modules)